components/brave_new_tab_ui/storage/new_tab_storage.ts

```diff
@@ -75,7 +75,7 @@ export const defaultState: NewTab.State = {
     dismissedNotifications: [],
     rewardsEnabled: false,
     enabledAds: false,
-    needsBrowserUpdateToSeeAds: false,
+    needsBrowserUpdateToViewAds: false,
```

Comment:
Updating the `rewardsState` schema for the NTP is dangerous, since there's no versioning and an old version of the schema may be stored in `localStorage`. However, since we are renaming a `boolean` property we *should* be OK here (since `undefined` will usually work in place of `false`).

Reply:
I will make sure we sanity test before merging. Thanks